main.py

```diff
@@ -320,6 +329,7 @@ def hotkey(self, event):
             self.frame_canvas.clear_last_label()
         elif event.char == "r":
             self.frame_canvas.rollingover_label = not self.frame_canvas.rollingover_label
+            self.mode_label.config(text="Mode: " + "rollover") if self.frame_canvas.rollingover_label == True else self.mode_label.config(text="Mode: " + "editing")
```

Comment:
Maybe the text can just be:

self.mode_label.config(text="Rolling Labels: " + str(self.frame_canvas.rollingover_label))